config/services.yml

```diff
@@ -0,0 +1,63 @@
+services:
+    _defaults:
+         autowire: true
+         autoconfigure: true
+         public: true
+
+    # General components
+
+    TwigYard\Component\AppState: ~
```

Comment:
I think we do not need to register the AppState

Reply:
Fixed